### PR TITLE
[4.0] Do NOT force jQuery.js and Bootstrap.js in the backend template

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue # .

### Summary of Changes
- Removed the dictated jQuery.js and Bootstrap.js from the backend template.
- Reason: almost ALL interactive components for Bootstrap are held by a PHP helper (`HTMLHelper::_('bootstrap.something'...)`) therefor there is zero benefit forcing these scripts in the index.php of the template. Also we've (as the Javascript team) put a huge amount of work to convert everything to vanilla javascript and quite frankly by forcing jQuery always makes all that work redundant.

### Testing Instructions
Check that all pages in the backend still work

### Expected result
No change

### Documentation Changes Required
No. jQuery and Bootstrap.js are still supported and can be loaded whenever they're needed but they're not forced anymore

### CODE REVIEW

File: administrator/modules/mod_feed/tmpl/default.php [hunk 1]
```diff
@@ -12,6 +12,9 @@
 use Joomla\CMS\Factory;
 use Joomla\CMS\Filter\OutputFilter;
 use Joomla\CMS\Language\Text;
+use Joomla\CMS\HTML\HTMLHelper;
```
Comment (Quy) [inline administrator/modules/mod_feed/tmpl/default.php:14]: Please sort in alpha order.

**Comment by wilsonge** [comment]
Just for the record *IF* we move to BS5 before stable (and therefore don't have to load jquery) I might reconsider this and revert back to loading bs5.js with the BACKEND template (not the frontend if/when we sort that out)

**Comment by dgrammatiko** [comment]
@wilsonge but the work on the custom elements was **primarily** to fix the a11y issues of Bootstrap. The vanilla js was a side effect. So falling back to BS is just blindly erase all the hard work done on the accessibility so far...

**Comment by dgrammatiko** [comment]
@Fedik did you just undid this PR with the changes in the assets manager?

**Comment by Fedik** [comment]
what changes? :)
anything can be, there was many changes

**Comment by dgrammatiko** [comment]
You switched back to **forcing** BS js on the index.php

**Comment by Fedik** [comment]
you mean changes in joomla.asset.json?
well, I did not forcing anything :neckbeard: 
there was 1000 merge conflicts in that file, that can accidentally mix the changes

**Comment by dgrammatiko** [comment]
I know, but basically you reverted this PR

**Pull request merged**